lib/Jsrt/JsrtRuntime.cpp

```diff
@@ -27,11 +27,17 @@ JsrtRuntime::JsrtRuntime(ThreadContext * threadContext, bool useIdle, bool dispa
 #ifdef ENABLE_DEBUG_CONFIG_OPTIONS
     serializeByteCodeForLibrary = false;
 #endif
+    this->jsrtDebugManager = nullptr;
 }
 
 JsrtRuntime::~JsrtRuntime()
 {
     HeapDelete(allocationPolicyManager);
+    if (this->jsrtDebugManager != nullptr)
```

Comment:
done

---
In reply to: [62747882](https://github.com/Microsoft/ChakraCore/pull/926#discussion_r62747882) [](ancestors = 62747882)